src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/controllers/SubjectAccessRequestController.kt

```diff
@@ -0,0 +1,108 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.controllers
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.SubjectAccessRequestContent
+import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.services.SubjectAccessRequestService
+import java.time.LocalDate
+
+/**
+ * Prisoners have the right to access and receive a copy of their personal data and other supplementary information.
+ *
+ * This is commonly referred to as a subject access request or ‘SAR’.
+ */
+@RestController
+@Tag(name = "Subject Access Request")
+@PreAuthorize("hasRole('SAR_DATA_ACCESS')")
+@RequestMapping("/subject-access-request", produces = [MediaType.APPLICATION_JSON_VALUE])
+class SubjectAccessRequestController(private val service: SubjectAccessRequestService) {
+
+  @GetMapping
+  @Operation(summary = "Provides content for a prisoner to satisfy the needs of a subject access request on their behalf")
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Request successfully processed - content found",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = SubjectAccessRequestContent::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "204",
+        description = "Request successfully processed - no content found",
+      ),
+      ApiResponse(
+        responseCode = "209",
```

Comment:
209? 404 surely 😆 (yes I know this wasn't your call)

Reply:
`418` seems more appropriate